samples/unit-tests/data/csv/demo.js

```diff
@@ -713,3 +685,27 @@ QUnit.test('Dot date format', function (assert) {
         }
     });
 });
+
+QUnit.test(
+    'Hex values are not treated as comments (#13283).',
+    function (assert) {
+        var chart = Highcharts.chart('container', {
+            chart: {
+                type: 'bar'
+            },
+            data: {
+                csv:
+                    'x;y;color\n' +
+                    '0;1;#F0F\n',
+
+                itemDelimiter: ';',
+                seriesMapping: [{
+                    color: 2
+                }]
+            }
+        });
+
+        var actualColor = chart.series[0].data[0].options.color;
+        assert.strictEqual(actualColor, '#F0F', 'Color must be a hex value');
```

Comment:
Not a big deal, but our general rule of thumb is that assertion descriptions should contain the word "should". We may add a pre-commit rule for it in the future.